src/main/webapp/WEB-INF/tags/instructor/course/statusMessage.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="Status Message of Course Page" %>
+<%@ tag import="teammates.common.util.Const" %>
+<br>
+<jsp:include page="<%=Const.ViewURIs.STATUS_MESSAGE%>" />
+<br>
```

Comment:
Wouldn't it be better to place the `<br>` wherever this tag is being called and use the tag equivalent of `STATUS_MESSAGE`?